website_customer_also_purchased/views/website_customer_purchased.xml

```diff
@@ -3,23 +3,13 @@
     <data>
     	<template id="customer_purchased" inherit_id="website_sale.product" customize_show="True" name="Other Customers Purchased">
 		  <xpath expr="//div[@id='product_full_description']" position="after">
-		      <div class="container mt32" t-if="product.customer_purchased">
-		          <h3>Other Customers Purchased:</h3>
-		          <div class='row mt16' style="margin-left: 15px !important;">
-		          <t t-foreach="product.customer_purchased" t-as="purchased_product">
-		              <div class='col-md-2 thumbnail' style='width: 170px; margin-right: 16px;'>
-		                  <div class='mt16 text-center'>
-		                      <span t-field="purchased_product.image_small" t-field-options='{"widget": "image", "class": "img-rounded shadow" }'/>
-		                      <h5>
-		                          <a t-attf-href="/shop/product/#{ slug(purchased_product) }" style="display: block">
-		                              <span t-field='purchased_product.name' style="display: block"/>
-		                          </a>
-		                      </h5>
-		                  </div>
-		              </div>
-		          </t>
-		          </div>
-		      </div>
+                  <div class="container mt32">
+                      <h3>Other Customers Purchased:</h3>
+                      <div class='row mt16' style="margin-left: 15px !important;">
```

Comment:
there is no need to add `!important` please use a boostrap padding class as `ml` http://aslanbakan.com/en/blog/less-space-responsive-css-margin-and-padding-helper-classes/